index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+  <style>
+    @media screen and (max-width: 740px) {
+      body {
+        background-color: lightgreen;
+      }
+    }
+  </style>
+</head>
+
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+ 
+  <section id="welcome-section">
+    <h1>Hello world, we are software developers</h1>
+    <h1>welcome to the page</h1>
+
+  </section>
+
+  <section id="projects">
+    <div class="project-tile">
+      Our First Project
+    </div>
+    <a href="https://google.com">First Project</a>
+  </section>
+  <nav id="navbar">
+    <a href="#welcome-section">Welcome Section</a>
+  </nav>
```

Comment:
Please surround these parts in the header tag. A header tag is a semantic tag and it creates meaning for your page structure.

Reply:
- [Optional] Kindly, uses a couple of emojis and/or gifs in the review. 👍
                    They make the code review more human-friendly 👽🤖.